pkg/controller/petset/pet.go

```diff
@@ -249,7 +251,8 @@ func (p *apiServerPetClient) SyncPVCs(pet *pcb) error {
 	for i, pvc := range pet.pvcs {
 		_, exists, err := p.getPVC(pvc.Name, pet.parent.Namespace)
 		if !exists {
-			if err := p.createPVC(&pet.pvcs[i]); err != nil {
+			var err error
+			if err = p.createPVC(&pet.pvcs[i]); err != nil {
 				errMsg += fmt.Sprintf("Failed to create %v: %v", pvc.Name, err)
 			}
 			p.event(pet.parent, "Create", fmt.Sprintf("pvc: %v", pvc.Name), err)
```

Comment:
I'm not sure I understand. I think if we did that, we would no longer accumulate the "Failed to create..." errors into the errMsg variable that we return from the function.